### DISCUSSION TITLE
I can't run  example code-templates

### DISCUSSION THREAD

**Discussion by Xc1Ym** [projectdiscovery/nuclei#4438] [Q&A] [answered]
### Nuclei version:
v3.0.4 windows

### Current Behavior:

I want to test this example from <https://github.com/projectdiscovery/nuclei/blob/main/integration_tests/protocols/code/ps1-snippet.yaml>
but I can't run it
![image](https://github.com/projectdiscovery/nuclei/assets/29765332/078243d2-5ff7-49ec-a5d3-d5dc796e7e68)

**Comment by ehsandeep** [comment, marked as answer]
@Xc1Ym 1) code template needs to signed before running using `-sign` option, 2) you need to additionally use `-code` option to enable loading code template.
